src/redux/README.md

```diff
@@ -42,6 +42,12 @@
         [_id]: { },
       }
     },
+    volumes: {
+      mapById: { volumes... },
+      clusterMapById: {
+        [clusterId]: 'volumeId',
+      },
+    },
     statuses: {
```

Comment:
This might have gotten slipped in when Tristan was talking about the volume work and react/redux while we were at KHQ.